restaurants.json

```diff
@@ -1,62 +0,0 @@
-[
-  {
-    "id": "ramuntos",
-    "name": "Ramuntos",
-    "address": "102 Strongs Ave. Rutland, VT",
-    "latitude": "43.601190",
-    "longitude": "-72.973275",
-    "phone number": "(802)747-9980",
-    "hours": "M-Thurs: 11AM-9PM F-Sat: 11AM-10PM Sun: 12PM-9PM",
-    "notes": "Sells single slices. Always has a 2 cheese slice special with a drink for ~ $5.00"
-  },
-  {
-    "id": "johnny-boys",
-    "name": "Johnny Boys Pancake House",
-    "address": "182 US Rt. 4 Rutland, VT",
-    "latitude": "43.622554",
-    "longitude": "-72.952676",
-    "phone number": "(802)855-8043",
-    "hours": "7AM-2PM",
-    "notes": "Great brunch spot. Always busy."
-  },
```

Comment:
Having an api end point for each individual restaurant as well as one large one with every restaurant is a little redundant. we could either use one large one and parse the information to find the data that we are looking for, or we can use smaller ones and only fetch for the specific ones that we need.